app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -386,6 +386,19 @@ class Settings(
             0,
         )
 
+    fun addClearBrowsingSessions(count: Int) {
```

Comment:
Documentation would be highly useful for this `Settings` methods which are often read by developers. 
It would help easily identifying the particular feature and scenario which each setting supports.

Reply:
Done